pom.xml

```diff
@@ -76,14 +76,14 @@
         <jdk.source>17</jdk.source>
         <jdk.target>17</jdk.target>
         <commons-collections4.version>4.4</commons-collections4.version>
-        <commons-io.version>2.11.0</commons-io.version>
-        <datatables-buttons.version>1.6.1</datatables-buttons.version>
+        <commons-io.version>2.13.0</commons-io.version>
+        <datatables-buttons.version>2.4.1</datatables-buttons.version>
         <datatables.version>1.10.24</datatables.version>
-        <font-awesome.version>4.7.0</font-awesome.version>
-        <jquery.version>3.6.4</jquery.version>
+        <font-awesome.version>6.4.0</font-awesome.version>
+        <jquery.version>3.7.0</jquery.version>
 
-        <lms-canvas-rivet.version>5.2.3.1_1</lms-canvas-rivet.version>
-        <lms-embedded-services.version>5.2.7</lms-embedded-services.version>
+        <lms-canvas-rivet.version>5.2.5.1</lms-canvas-rivet.version>
+        <lms-embedded-services.version>5.2.11</lms-embedded-services.version>
```

Comment:
Waiting for latest embedded services release so that this can go to that version